templates/environment/cf.yml

```diff
@@ -63,14 +63,14 @@ Resources:
       InstanceTenancy: default
       Tags:
         - Key: Name
-          Value: !Sub 'ecs-${ProjectName}-${EnvironmentName}'
+          Value: !Sub 'ecs-${AppName}-${EnvironmentName}'
```

Comment:
```suggestion
          Value: !Sub 'copilot-${AppName}-${EnvironmentName}'
```
There are a few other places below that'd need to change to `copilot` too for the `Name` tag!

Reply:
Or more general, anywhere with `ecs-cli` or `ecs` in it

Reply:
AWS::COPILOT::Cluster